iproute2.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: iproute2
+  version: "6.2.0"
+  epoch: 0
+  description: IP Routing Utilities
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - bison
+      - flex
+      - bash
+      - iptables-dev
+      - elfutils-dev
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://kernel.org/pub/linux/utils/net/iproute2/iproute2-${{package.version}}.tar.xz
```

Comment:
Would it be advantageous to use the git repo instead here? (I don't think I'm fully aware of the tradeoffs that would determine the answer to this. But I have a soft intuition that traceability into source history will be helpful over time)

(https://github.com/shemminger/iproute2)

Reply:
I looked at this but wasn't terribly confident it was really maintained or official. I'm happy to switch if it's real.